src/ray/core_worker/core_worker.cc

```diff
@@ -103,6 +103,17 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
   gcs_client_ = std::make_shared<gcs::RedisGcsClient>(gcs_options);
   RAY_CHECK_OK(gcs_client_->Connect(io_service_));
 
+  // Subscribe to all unexpected worker failure notifications from the local
+  // and remote raylets.
```

Comment:
This doesn't include a notification if the node the worker on fails unexpected right?

Reply:
That's right, I'll update the comment.